charts/keycloakx/README.md

```diff
@@ -174,7 +173,7 @@ The following table lists the configurable parameters of the Keycloak-X chart an
 | `proxy.mode`                                 | The configured proxy mode                                                                                                                                                                                                                                                         | `edge`                                                                                                                                |
 | `http.relativePath`                          | The relative http path (context-path)                                                                                                                                                                                                                                             | `/auth`                                                                                                                               |
 | `metrics.enabled`                            | If `true` then the metrics endpoint is exposed                                                                                                                                                                                                                                    | `true`                                                                                                                                |
-| `health.enabled`                            | If `true` then the health endpoint is exposed                                                                                                                                                                                                                                    | `true`                                                                                                                                |
+| `health.enabled`                            | If `true` then the health endpoint is exposed. If the `readinessProbe` is is needed `metrics.enable` must be `true`.                                                                                                                                                                                                                                    | `true`                                                                                                                                |
```

Comment:
> If the `readinessProbe` is is needed `metrics.enable` must be `true`
@Footur are you sure this is true? Asking since Keycloak Operator creates readinessProbe with metrics disabled. For some reason, by default, metrics are disabled. Currently I have metrics disabled but defined readinessProbe. After this change I will be forced to enable the metrics to not lose the probe.